pkg/render/kube-controllers.go

```diff
@@ -390,6 +391,19 @@ func (c *kubeControllersComponent) controllersDeployment() *apps.Deployment {
 				},
 			},
 		},
+		LivenessProbe: &v1.Probe{
```

Comment:
I think the change adding `-l` to kube-controllers needs to be merged to the Enterprise version before merging this.

Reply:
Done :)